### ISSUE TITLE
Display issue

### ISSUE THREAD

**Issue by shtrom** [nextcloud/deck#346]
### Steps to reproduce
I'm not quite sure how to reproduce, I've been seeing this issue for a while:

![screenshot from 2017-11-23 21-20-13](https://user-images.githubusercontent.com/160280/33168175-28a38466-d094-11e7-96ba-eda071f88fe8.png)

### Expected behaviour
There should be three column using the full width of the browser.

### Actual behaviour

There is a gap on the right-hand side, and everything is displayed, with poor styling, in a single colume

### Server configuration

**Operating system**: OpenBSD 6.1

**Web server:** Apache 2.4

**Database:** MySQL

**PHP version:** 5.6

**Nextcloud version:** (see Nextcloud admin page) 12.0.0

**Where did you install Nextcloud from:** OpenBSD ports

**Signing status:** A couple of failed signatures due to the OpenBSD packaging, but it used to work fine even with those

<details>

```
Technical information
=====================
The following list covers which files have failed the integrity check. Please read
the previous linked documentation to learn more about the errors and how to fix
them.

Results
=======
- core
	- INVALID_HASH
		- occ
		- version.php
- updatenotification
	- INVALID_HASH
		- appinfo/info.xml

Raw output
==========
Array
(
    [core] => Array
        (
            [INVALID_HASH] => Array
                (
                    [occ] => Array
                        (
                            [expected] => 7e3fce0d7b5c20a7775ed1b548cb2e29bed078d3ca77b01a83d438f671b3d473147d4e8217d2084e17b6fe23a18ba258b11ba60106e23381f1e2889ce14971c4
                            [current] => 7693eb89c0bc218712d68ec58599efa46e5c3729814e2aad16bf2c0079be7ae1909f072ead7889883c0a89b6c51570800d9e8a71f35866cb4e0c47aeaa5a4b2b
                        )

                    [version.php] => Array
                        (
                            [expected] => 63bdc8487d37c7ba6976664d7996ed0d41d770755605b59c4cff35b509a797c3ca6f44670a8384b155430c8bba36e6fe04e3eca97f3afe2d4b2d9eb051a1fdb6
                            [current] => 79a890d671fa415159011f73ce5f4916aae4bb8cb07573f3590200897deacaeb2d6fd29ed73e9ec131e49a7f3559e3b22a95f281b558fda2fb57bf2fa44ea30c
                        )

                )

        )

    [updatenotification] => Array
        (
            [INVALID_HASH] => Array
                (
                    [appinfo/info.xml] => Array
                        (
                            [expected] => 9339cdd38bde362ed4700416459fb3ea192f69fe3d78f38d26b11e6316ca6cfa65393b4314994f07e28c4ebc0277136c21d449258aa65be5a32dcddc8b94e25d
                            [current] => 775fecda2defffb8bbb000f9c4890f8b5519c677d550d4fc4865bb7ca32c3439bc62f665a3a9c8f1eb4fd56a7d3352aabca34d8f1a4e50ae256b50eef8790852
                        )

                )

        )

)
```
</details>
**List of activated apps:**
<details>

```
Enabled:
  - bookmarks: 0.10.1
  - bruteforcesettings: 1.0.2
  - calendar: 1.5.6
  - contacts: 2.0.1
  - dav: 1.3.0
  - deck: 0.2.7
  - federatedfilesharing: 1.2.0
  - federation: 1.2.0
  - files: 1.7.2
  - files_mv: 0.8.2
  - files_pdfviewer: 1.1.1
  - files_sharing: 1.4.0
  - files_texteditor: 2.4.1
  - files_trashbin: 1.2.0
  - files_versions: 1.5.0
  - files_videoplayer: 1.1.0
  - gallery: 17.0.0
  - logreader: 2.0.0
  - lookup_server_connector: 1.0.0
  - mail: 0.7.4
  - mozilla_sync: true
  - news: 11.0.5
  - nextcloud_announcements: 1.1
  - notes: 2.3.1
  - notifications: 2.0.0
  - oauth2: 1.0.5
  - ocsms: 1.12.1
  - ojsxc: 3.3.1
  - password_policy: 1.2.2
  - provisioning_api: 1.2.0
  - sensorlogger: 0.0.4
  - serverinfo: 1.2.0
  - sharebymail: 1.2.0
  - socialsharing_email: 1.0.1
  - spreed: 2.0.1
  - survey_client: 1.0.0
  - tasks: 0.9.5
  - theming: 1.3.0
  - twofactor_backupcodes: 1.1.1
  - twofactor_u2f: 1.4.0
  - updatenotification: 1.2.0
  - user_ldap: 1.2.1
  - workflowengine: 1.2.0
Disabled:
  - activity
  - admin_audit
  - comments
  - encryption
  - files_external
  - firstrunwizard
  - systemtags
  - user_external
```
</details>

**Nextcloud configuration:**
<details>

```
{
    "system": {
        "instanceid": "ocbyhsif2tar",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "cloud.example.net"
        ],
        "datadirectory": "\/sites\/example.net\/cloud\/data",
        "dbtype": "mysql",
        "dbname": "nextcloud",
        "dbhost": "localhost",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "version": "12.0.0.29",
        "logtimezone": "UTC",
        "installed": true,
        "mail_smtpmode": "smtp",
        "mail_smtphost": "127.0.0.1",
        "mail_smtpport": "25",
        "mail_smtptimeout": 10,
        "mail_from_address": "cloud",
        "mail_domain": "example.net",
        "appstoreenabled": true,
        "apps_paths": [
            {
                "path": "\/sites\/example.net\/cloud\/apps",
                "url": "\/local\/apps",
                "writable": true
            },
            {
                "path": "\/nextcloud\/apps",
                "url": "\/apps",
                "writable": false
            }
        ],
        "ldapIgnoreNamingRules": false,
        "ldapProviderFactory": "\\OCA\\User_LDAP\\LDAPProviderFactory",
        "loglevel": 2,
        "maintenance": false,
        "memcache.local": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "localhost",
            "port": 6379
        },
        "lost_password_link": "https:\/\/www.example.net\/passwd",
        "overwrite.cli.url": "https:\/\/cloud.example.net",
        "htaccess.RewriteBase": "\/",
        "mysql.utf8mb4": true,
        "theme": ""
    },
    "apps": {
        "activity": {
            "enabled": "no",
            "installed_version": "2.5.2",
            "types": "filesystem"
        },
        "admin_dependencies_chk": {
            "enabled": "no",
            "installed_version": "0.02",
            "ocsid": "166041",
            "types": ""
        },
        "admin_migrate": {
            "enabled": "no",
            "installed_version": "0.1",
            "types": ""
        },
        "admin_notifications": {
            "enabled": "no",
            "installed_version": "1.0.0",
            "types": "logging"
        },
        "appframework": {
            "enabled": "no",
            "installed_version": "0.104",
            "ocsid": "162405",
            "types": ""
        },
        "backgroundjob": {
            "lastjob": "1720"
        },
        "bookmarks": {
            "enabled": "yes",
            "installed_version": "0.10.1",
            "types": ""
        },
        "bruteforcesettings": {
            "enabled": "yes",
            "installed_version": "1.0.2",
            "types": ""
        },
        "calendar": {
            "enabled": "yes",
            "installed_version": "1.5.6",
            "types": ""
        },
        "calendarplus": {
            "enabled": "no",
            "installed_version": "1.0.2",
            "ocsid": "170560",
            "types": ""
        },
        "comments": {
            "enabled": "no",
            "installed_version": "0.2",
            "types": "logging"
        },
        "configreport": {
            "enabled": "no",
            "installed_version": "0.1.1",
            "types": "filesystem"
        },
        "contacts": {
            "backend_ldap": "false",
            "enabled": "yes",
            "installed_version": "2.0.1",
            "ocsid": "168708",
            "types": ""
        },
        "core": {
            "OC_Channel": "stable",
            "backgroundjobs_mode": "cron",
            "backgroundjobs_step": "regular_tasks",
            "backgroundjobs_task": "",
            "default_encryption_module": "OC_DEFAULT_MODULE",
            "global_cache_gc_lastrun": "1436400016",
            "installed.bundles": "[\"CoreBundle\"]",
            "installedat": "1384525652.3607",
            "lastcron": "1511432126",
            "lastupdateResult": "[]",
            "lastupdatedat": "1511432537",
            "moveavatarsdone": "yes",
            "oc.integritycheck.checker": "{\"core\":{\"INVALID_HASH\":{\"occ\":{\"expected\":\"7e3fce0d7b5c20a7775ed1b548cb2e29bed078d3ca77b01a83d438f671b3d473147d4e8217d2084e17b6fe23a18ba258b11ba60106e23381f1e2889ce14971c4\",\"current\":\"7693eb89c0bc218712d68ec58599efa46e5c3729814e2aad16bf2c0079be7ae1909f072ead7889883c0a89b6c51570800d9e8a71f35866cb4e0c47aeaa5a4b2b\"},\"version.php\":{\"expected\":\"63bdc8487d37c7ba6976664d7996ed0d41d770755605b59c4cff35b509a797c3ca6f44670a8384b155430c8bba36e6fe04e3eca97f3afe2d4b2d9eb051a1fdb6\",\"current\":\"79a890d671fa415159011f73ce5f4916aae4bb8cb07573f3590200897deacaeb2d6fd29ed73e9ec131e49a7f3559e3b22a95f281b558fda2fb57bf2fa44ea30c\"}}},\"updatenotification\":{\"INVALID_HASH\":{\"appinfo\\\/info.xml\":{\"expected\":\"9339cdd38bde362ed4700416459fb3ea192f69fe3d78f38d26b11e6316ca6cfa65393b4314994f07e28c4ebc0277136c21d449258aa65be5a32dcddc8b94e25d\",\"current\":\"775fecda2defffb8bbb000f9c4890f8b5519c677d550d4fc4865bb7ca32c3439bc62f665a3a9c8f1eb4fd56a7d3352aabca34d8f1a4e50ae256b50eef8790852\"}}}}",
            "previewsCleanedUp": "1",
            "public_caldav": "calendar\/share.php",
            "public_calendar": "calendar\/share.php",
            "public_calendarplus": "calendarplus\/share.php",
            "public_calendarplusevent": "calendarplus\/shareevent.php",
            "public_documents": "documents\/public.php",
            "public_files": "files_sharing\/public.php",
            "public_gallery": "gallery\/public.php",
            "public_todo": "tasksplus\/sharetodo.php",
            "public_webdav": "dav\/appinfo\/v1\/publicwebdav.php",
            "remote_ampache": "media\/remote.php",
            "remote_caldav": "dav\/appinfo\/v1\/caldav.php",
            "remote_calendar": "dav\/appinfo\/v1\/caldav.php",
            "remote_calendarplus": "calendarplus\/appinfo\/remote.php",
            "remote_carddav": "dav\/appinfo\/v1\/carddav.php",
            "remote_contacts": "dav\/appinfo\/v1\/carddav.php",
            "remote_contactthumbnail": "contacts\/thumbnail.php",
            "remote_core.css": "\/core\/minimizer.php",
            "remote_core.js": "\/core\/minimizer.php",
            "remote_dav": "dav\/appinfo\/v2\/remote.php",
            "remote_files": "dav\/appinfo\/v1\/webdav.php",
            "remote_filesync": "files\/appinfo\/filesync.php",
            "remote_mozilla_sync": "mozilla_sync\/appinfo\/remote.php",
            "remote_webdav": "dav\/appinfo\/v1\/webdav.php",
            "repairlegacystoragesdone": "yes",
            "scss.variables": "0642e60a2b959a74947b4cc5bc89ffc0",
            "shareapi_allow_mail_notification": "yes",
            "shareapi_allow_public_notification": "yes",
            "shareapi_exclude_groups": "no",
            "shareapi_only_share_with_group_members": "yes",
            "vendor": "nextcloud"
        },
        "dav": {
            "OCA\\DAV\\Migration\\ValueFixInsert_ran": "true",
            "buildCalendarSearchIndex": "yes",
            "enabled": "yes",
            "installed_version": "1.3.0",
            "types": "filesystem"
        },
        "deck": {
            "enabled": "yes",
            "installed_version": "0.2.7",
            "types": ""
        },
        "delete_confirmation": {
            "enabled": "no",
            "installed_version": "1.1.0",
            "types": ""
        },
        "direct_menu": {
            "enabled": "no",
            "installed_version": "0.10.2",
            "ocsid": "169148",
            "types": ""
        },
        "documents": {
            "enabled": "no",
            "installed_version": "0.15.0",
            "ocsid": "168711",
            "types": ""
        },
        "encryption": {
            "enabled": "no",
            "installed_version": "1.2.0",
            "masterKeyId": "master_06a2a28f",
            "publicShareKeyId": "pubShare_8336aaaf",
            "recoveryKeyId": "recoveryKey_06a2a28f",
            "types": "filesystem"
        },
        "external": {
            "enabled": "no",
            "installed_version": "",
            "ocsid": "166046",
            "sites": "",
            "types": ""
        },
        "federatedfilesharing": {
            "enabled": "yes",
            "installed_version": "1.2.0",
            "types": ""
        },
        "federation": {
            "enabled": "yes",
            "installed_version": "1.2.0",
            "types": "authentication"
        },
        "files": {
            "backgroundwatcher_previous_file": "10821",
            "backgroundwatcher_previous_folder": "10538",
            "cronjob_scan_files": "500",
            "enabled": "yes",
            "installed_version": "1.7.2",
            "types": "filesystem"
        },
        "files_external": {
            "allow_user_mounting": "yes",
            "enabled": "no",
            "installed_version": "0.5.2",
            "ocsid": "166048",
            "types": "filesystem",
            "user_mounting_backends": "\\OC\\Files\\Storage\\AmazonS3,\\OC\\Files\\Storage\\Dropbox,\\OC\\Files\\Storage\\FTP,\\OC\\Files\\Storage\\Google,\\OC\\Files\\Storage\\Swift,\\OC\\Files\\Storage\\OwnCloud,\\OC\\Files\\Storage\\SFTP,\\OC\\Files\\Storage\\DAV"
        },
        "files_imageviewer": {
            "enabled": "no",
            "installed_version": "1.0",
            "types": ""
        },
        "files_locking": {
            "enabled": "no",
            "installed_version": "",
            "types": "filesystem"
        },
        "files_mv": {
            "enabled": "yes",
            "installed_version": "0.8.2",
            "ocsid": "150271",
            "types": "filesystem"
        },
        "files_odfviewer": {
            "enabled": "no",
            "installed_version": "0.1",
            "types": ""
        },
        "files_pdfviewer": {
            "enabled": "yes",
            "installed_version": "1.1.1",
            "ocsid": "166049",
            "types": ""
        },
        "files_sharing": {
            "enabled": "yes",
            "installed_version": "1.4.0",
            "types": "filesystem"
        },
        "files_texteditor": {
            "enabled": "yes",
            "installed_version": "2.4.1",
            "ocsid": "166051",
            "types": ""
        },
        "files_trashbin": {
            "enabled": "yes",
            "installed_version": "1.2.0",
            "types": "filesystem"
        },
        "files_versions": {
            "enabled": "yes",
            "installed_version": "1.5.0",
            "types": "filesystem"
        },
        "files_videoplayer": {
            "enabled": "yes",
            "installed_version": "1.1.0",
            "types": ""
        },
        "files_videoviewer": {
            "enabled": "no",
            "installed_version": "0.1.3",
            "ocsid": "166054",
            "types": ""
        },
        "firstrunwizard": {
            "enabled": "no",
            "installed_version": "1.1",
            "ocsid": "166055",
            "types": ""
        },
        "gallery": {
            "enabled": "yes",
            "installed_version": "17.0.0",
            "types": ""
        },
        "journal": {
            "enabled": "no",
            "installed_version": "0.3.2",
            "ocsid": "154637",
            "types": ""
        },
        "logreader": {
            "enabled": "yes",
            "installed_version": "2.0.0",
            "levels": "00111",
            "ocsid": "170871",
            "types": ""
        },
        "lookup_server_connector": {
            "enabled": "yes",
            "installed_version": "1.0.0",
            "types": "authentication"
        },
        "mail": {
            "enabled": "yes",
            "installed_version": "0.7.4",
            "signed": "true",
            "types": ""
        },
        "media": {
            "enabled": "no",
            "installed_version": "0.4.3",
            "types": ""
        },
        "mozilla_sync": {
            "authorized_group": null,
            "enabled": "yes",
            "installed_version": "",
            "ocsid": "161793",
            "types": ""
        },
        "news": {
            "enabled": "yes",
            "installed_version": "11.0.5",
            "ocsid": "168040",
            "types": ""
        },
        "nextant": {
            "installed_version": "1.0.6",
            "types": "filesystem"
        },
        "nextcloud_announcements": {
            "enabled": "yes",
            "installed_version": "1.1",
            "pub_date": "Sat, 10 Dec 2016 00:00:00 +0100",
            "types": "logging"
        },
        "notes": {
            "enabled": "yes",
            "installed_version": "2.3.1",
            "ocsid": "160567",
            "types": ""
        },
        "notifications": {
            "enabled": "yes",
            "installed_version": "2.0.0",
            "types": "logging"
        },
        "oauth2": {
            "enabled": "yes",
            "installed_version": "1.0.5",
            "types": "authentication"
        },
        "ocsms": {
            "enabled": "yes",
            "installed_version": "1.12.1",
            "ocsid": "167289",
            "types": ""
        },
        "ojsxc": {
            "apiSecret": "ZNA8r6pDOvBBbr9dF5aYmCw",
            "enabled": "yes",
            "installed_version": "3.3.1",
            "ocsid": "162257",
            "types": "prelogin"
        },
        "ownnote": {
            "enabled": "no",
            "installed_version": "1.05",
            "ocsid": "168512",
            "types": ""
        },
        "password_policy": {
            "enabled": "yes",
            "installed_version": "1.2.2",
            "types": ""
        },
        "provisioning_api": {
            "enabled": "no",
            "installed_version": "1.2.0",
            "types": "prevent_group_restriction"
        },
        "roundcube": {
            "enabled": "no",
            "installed_version": "2.5.4",
            "ocsid": "151523",
            "types": ""
        },
        "search_lucene": {
            "enabled": "no",
            "installed_version": "0.5.3",
            "ocsid": "168709",
            "types": "filesystem"
        },
        "sensorlogger": {
            "enabled": "yes",
            "installed_version": "0.0.4",
            "types": ""
        },
        "serverinfo": {
            "enabled": "yes",
            "installed_version": "1.2.0",
            "types": ""
        },
        "sharebymail": {
            "enabled": "yes",
            "installed_version": "1.2.0",
            "types": "filesystem"
        },
        "socialsharing_email": {
            "enabled": "yes",
            "installed_version": "1.0.1",
            "types": ""
        },
        "spreed": {
            "enabled": "yes",
            "installed_version": "2.0.1",
            "types": "prevent_group_restriction"
        },
        "survey_client": {
            "enabled": "yes",
            "installed_version": "1.0.0",
            "last_report": "{\"id\":\"ocbyhsif2tar\",\"items\":[[\"server\",\"version\",\"12.0.0.29\"],[\"server\",\"code\",\"other\"],[\"server\",\"enable_avatars\",\"yes\"],[\"server\",\"enable_previews\",\"yes\"],[\"server\",\"memcache.local\",\"\\\\OC\\\\Memcache\\\\Redis\"],[\"server\",\"memcache.distributed\",\"none\"],[\"server\",\"asset-pipeline.enabled\",\"no\"],[\"server\",\"filelocking.enabled\",\"yes\"],[\"server\",\"memcache.locking\",\"none\"],[\"server\",\"debug\",\"no\"],[\"server\",\"cron\",\"cron\"],[\"php\",\"version\",\"5.6.30\"],[\"php\",\"memory_limit\",134217728],[\"php\",\"max_execution_time\",0],[\"php\",\"upload_max_filesize\",2097152],[\"database\",\"type\",\"mysql\"],[\"database\",\"version\",\"10.0.30\"],[\"database\",\"size\",170573824],[\"apps\",158433,\"disabled\"],[\"apps\",\"activity\",\"disabled\"],[\"apps\",\"admin_dependencies_chk\",\"disabled\"],[\"apps\",\"admin_migrate\",\"disabled\"],[\"apps\",\"admin_notifications\",\"disabled\"],[\"apps\",\"appframework\",\"disabled\"],[\"apps\",\"bookmarks\",\"0.10.1\"],[\"apps\",\"bruteforcesettings\",\"1.0.2\"],[\"apps\",\"calendar\",\"1.5.4\"],[\"apps\",\"calendarplus\",\"disabled\"],[\"apps\",\"comments\",\"disabled\"],[\"apps\",\"configreport\",\"disabled\"],[\"apps\",\"contacts\",\"1.5.3\"],[\"apps\",\"dav\",\"1.3.0\"],[\"apps\",\"deck\",\"0.2.3\"],[\"apps\",\"delete_confirmation\",\"disabled\"],[\"apps\",\"direct_menu\",\"disabled\"],[\"apps\",\"documents\",\"disabled\"],[\"apps\",\"encryption\",\"disabled\"],[\"apps\",\"external\",\"disabled\"],[\"apps\",\"federatedfilesharing\",\"1.2.0\"],[\"apps\",\"federation\",\"1.2.0\"],[\"apps\",\"files\",\"1.7.2\"],[\"apps\",\"files_external\",\"disabled\"],[\"apps\",\"files_imageviewer\",\"disabled\"],[\"apps\",\"files_locking\",\"disabled\"],[\"apps\",\"files_mv\",\"0.8.2\"],[\"apps\",\"files_odfviewer\",\"disabled\"],[\"apps\",\"files_pdfviewer\",\"1.1.1\"],[\"apps\",\"files_sharing\",\"1.4.0\"],[\"apps\",\"files_texteditor\",\"2.4.1\"],[\"apps\",\"files_trashbin\",\"1.2.0\"],[\"apps\",\"files_versions\",\"1.5.0\"],[\"apps\",\"files_videoplayer\",\"1.1.0\"],[\"apps\",\"files_videoviewer\",\"disabled\"],[\"apps\",\"firstrunwizard\",\"disabled\"],[\"apps\",\"gallery\",\"17.0.0\"],[\"apps\",\"journal\",\"disabled\"],[\"apps\",\"logreader\",\"2.0.0\"],[\"apps\",\"lookup_server_connector\",\"1.0.0\"],[\"apps\",\"mail\",\"0.7.3\"],[\"apps\",\"media\",\"disabled\"],[\"apps\",\"mozilla_sync\",\"\"],[\"apps\",\"news\",\"11.0.5\"],[\"apps\",\"nextant\",\"1.0.6\"],[\"apps\",\"nextcloud_announcements\",\"1.1\"],[\"apps\",\"notes\",\"2.3.1\"],[\"apps\",\"notifications\",\"2.0.0\"],[\"apps\",\"oauth2\",\"1.0.5\"],[\"apps\",\"ojsxc\",\"3.3.0\"],[\"apps\",\"ownnote\",\"disabled\"],[\"apps\",\"password_policy\",\"1.2.2\"],[\"apps\",\"provisioning_api\",\"disabled\"],[\"apps\",\"roundcube\",\"disabled\"],[\"apps\",\"search_lucene\",\"disabled\"],[\"apps\",\"sensorlogger\",\"0.0.4\"],[\"apps\",\"serverinfo\",\"1.2.0\"],[\"apps\",\"sharebymail\",\"1.2.0\"],[\"apps\",\"socialsharing_email\",\"1.0.1\"],[\"apps\",\"spreed\",\"2.0.1\"],[\"apps\",\"survey_client\",\"1.0.0\"],[\"apps\",\"systemtags\",\"disabled\"],[\"apps\",\"tal\",\"disabled\"],[\"apps\",\"tasks\",\"0.9.5\"],[\"apps\",\"tasksplus\",\"disabled\"],[\"apps\",\"templateeditor\",\"disabled\"],[\"apps\",\"theming\",\"1.3.0\"],[\"apps\",\"twofactor_backupcodes\",\"1.1.1\"],[\"apps\",\"twofactor_u2f\",\"1.3.3\"],[\"apps\",\"updatenotification\",\"1.2.0\"],[\"apps\",\"updater\",\"disabled\"],[\"apps\",\"user_external\",\"disabled\"],[\"apps\",\"user_ldap\",\"1.2.1\"],[\"apps\",\"user_migrate\",\"disabled\"],[\"apps\",\"user_otp\",\"disabled\"],[\"apps\",\"videos\",\"disabled\"],[\"apps\",\"workflowengine\",\"1.2.0\"],[\"stats\",\"num_files\",27405],[\"stats\",\"num_users\",5],[\"stats\",\"num_storages\",21],[\"stats\",\"num_storages_local\",2],[\"stats\",\"num_storages_home\",14],[\"stats\",\"num_storages_other\",5],[\"stats\",\"num_comments\",0],[\"stats\",\"num_comment_markers\",0],[\"stats\",\"num_systemtags\",0],[\"stats\",\"num_systemtags_mappings\",0],[\"files_sharing\",\"num_shares\",17],[\"files_sharing\",\"num_shares_user\",10],[\"files_sharing\",\"num_shares_groups\",1],[\"files_sharing\",\"num_shares_link\",6],[\"files_sharing\",\"num_shares_link_no_password\",6],[\"files_sharing\",\"num_fed_shares_sent\",0],[\"files_sharing\",\"num_fed_shares_received\",0],[\"files_sharing\",\"permissions_0_1\",\"1\"],[\"files_sharing\",\"permissions_3_1\",\"6\"],[\"files_sharing\",\"permissions_0_17\",\"3\"],[\"files_sharing\",\"permissions_0_19\",\"3\"],[\"files_sharing\",\"permissions_0_31\",\"3\"],[\"files_sharing\",\"permissions_1_31\",\"1\"],[\"encryption\",\"enabled\",\"no\"],[\"encryption\",\"default_module\",\"yes\"]]}",
            "last_sent": "1506752121",
            "types": ""
        },
        "systemtags": {
            "enabled": "no",
            "installed_version": "0.2",
            "types": "logging"
        },
        "tal": {
            "enabled": "no",
            "installed_version": "0.2.2",
            "ocsid": "154632",
            "types": ""
        },
        "tasks": {
            "enabled": "yes",
            "installed_version": "0.9.5",
            "ocsid": "164356",
            "signed": "true",
            "types": ""
        },
        "tasksplus": {
            "enabled": "no",
            "installed_version": "1.0.0",
            "ocsid": "170561",
            "types": ""
        },
        "templateeditor": {
            "enabled": "no",
            "installed_version": "0.2",
            "types": ""
        },
        "theming": {
            "backgroundMime": "image\/jpeg",
            "cachebuster": "8",
            "color": "#791DC9",
            "enabled": "yes",
            "installed_version": "1.3.0",
            "logoMime": "image\/png",
            "name": "Cloud",
            "slogan": "my own computer",
            "types": "logging",
            "url": "https:\/\/cloud.example.net"
        },
        "twofactor_backupcodes": {
            "enabled": "yes",
            "installed_version": "1.1.1",
            "types": ""
        },
        "twofactor_u2f": {
            "enabled": "yes",
            "installed_version": "1.4.0",
            "types": ""
        },
        "updatenotification": {
            "bookmarks": "0.10.1",
            "calendar": "1.5.6",
            "contacts": "2.0.1",
            "core": "9.1.6",
            "deck": "0.2.7",
            "documents": "0.13.1",
            "enabled": "yes",
            "installed_version": "1.2.0",
            "mail": "0.7.4",
            "mozilla_sync": "1.4",
            "news": "11.0.5",
            "notes": "2.3.1",
            "ojsxc": "3.3.1",
            "tasks": "0.9.4",
            "twofactor_u2f": "1.4.0",
            "types": "",
            "update_check_errors": "0"
        },
        "updater": {
            "enabled": "no",
            "installed_version": "0.6",
            "types": ""
        },
        "user_external": {
            "enabled": "no",
            "installed_version": "0.4",
            "ocsid": "166060",
            "types": "authentication,prelogin"
        },
        "user_ldap": {
            "cleanUpJobOffset": "0",
            "enabled": "yes",
            "enforce_home_folder_naming_rule": "",
            "has_memberof_filter_support": "",
            "home_folder_naming_rule": "attr:uid",
            "installed_version": "1.2.1",
            "last_jpegPhoto_lookup": "0",
            "ldap_agent_password": "***REMOVED SENSITIVE VALUE***",
            "ldap_attributes_for_group_search": "",
            "ldap_attributes_for_user_search": "",
            "ldap_backup_host": "",
            "ldap_backup_port": "",
            "ldap_base": "ou=people,dc=narf,dc=ssji,dc=net\nou=people,dc=narf,dc=ssji,dc=net",
            "ldap_base_groups": "ou=people,dc=narf,dc=ssji,dc=net",
            "ldap_base_users": "ou=people,dc=narf,dc=ssji,dc=net",
            "ldap_cache_ttl": "600",
            "ldap_configuration_active": "1",
            "ldap_default_ppolicy_dn": "",
            "ldap_display_name": "cn",
            "ldap_dn": "cn=lookup,dc=narf,dc=ssji,dc=net",
            "ldap_dynamic_group_member_url": "",
            "ldap_email_attr": "mail",
            "ldap_experienced_admin": "0",
            "ldap_expert_username_attr": "uid",
            "ldap_expert_uuid_group_attr": "",
            "ldap_expert_uuid_user_attr": "",
            "ldap_gid_number": "gidNumber",
            "ldap_group_display_name": "cn",
            "ldap_group_filter": "(&(|(objectclass=posixGroup)))",
            "ldap_group_filter_mode": "1",
            "ldap_group_member_assoc_attribute": "memberUid",
            "ldap_groupfilter_groups": "",
            "ldap_groupfilter_objectclass": "posixGroup",
            "ldap_host": "127.0.0.1",
            "ldap_login_filter": "(&(|(objectclass=inetOrgPerson))(uid=%uid))",
            "ldap_login_filter_mode": "1",
            "ldap_loginfilter_attributes": "",
            "ldap_loginfilter_email": "0",
            "ldap_loginfilter_username": "1",
            "ldap_nested_groups": "0",
            "ldap_override_main_server": "0",
            "ldap_paging_size": "500",
            "ldap_port": "389",
            "ldap_quota_attr": "",
            "ldap_quota_def": "",
            "ldap_tls": "0",
            "ldap_turn_off_cert_check": "0",
            "ldap_turn_on_pwd_change": "0",
            "ldap_user_display_name_2": "",
            "ldap_user_filter_mode": "1",
            "ldap_userfilter_groups": "",
            "ldap_userfilter_objectclass": "inetOrgPerson",
            "ldap_userlist_filter": "(|(objectclass=inetOrgPerson))",
            "types": "authentication",
            "use_memberof_to_detect_membership": "1"
        },
        "user_migrate": {
            "enabled": "no",
            "installed_version": "0.1",
            "types": ""
        },
        "user_otp": {
            "enabled": "no",
            "installed_version": "2.5",
            "types": "authentication"
        },
        "videos": {
            "enabled": "no",
            "installed_version": "1.0",
            "ocsid": "161584",
            "types": ""
        },
        "workflowengine": {
            "enabled": "yes",
            "installed_version": "1.2.0",
            "types": "filesystem"
        },
        "158433": {
            "enabled": "no"
        }
    }
}

```
</details>

**Are you using an external user-backend, if yes which one:** LDAP

### Client configuration
**Browser:** Firefox 57

**Operating system:** ArchLinux x86_64

#### Nextcloud log (data/nextcloud.log)
I only get 200s or other successful responses from the server.

#### Browser log
<details>

```
Content Security Policy: Ignoring duplicate source 'unsafe-inline'
Content Security Policy: Couldn’t parse invalid host 'unsafe-inline'
JQMIGRATE: Migrate is installed, version 1.4.0
core.js:7:542
Source map error: SyntaxError: JSON.parse: unexpected character at line 1 column 1 of the JSON data
Resource URL: https://cloud.narf.ssji.net/core/vendor/core.js?v=d5796c60944bd91c75c7195b15b97c9a-8
Source Map URL: ./dist/purify.min.js.map
[Learn More]
State changed to INITIATING
jsxc.js:183:9
State changed to PREVCONFOUND
jsxc.js:183:9
check master
jsxc.js:183:9
I am the slave.
jsxc.js:183:9
UI State changed to INITIATING
jsxc.js:183:9
TypeError: jsxc.xmpp.conn is null
```
</details>

**Comment by shtrom** [comment]
Same thing after clearing the whole browser's cache.

**Comment by shtrom** [comment]
Same thing after clearing the whole browser's cache.

**Comment by shtrom** [comment]
Ok, this is an issue with using apps with SCSS stylesheets in a different location. Details are in nextcould/contacts#442, but the fix from nextcloud/server#7257  works.
